README.md

```diff
@@ -55,16 +55,31 @@ This library supports the following Google Cloud Platform services with clients
 Quickstart
 ----------
 
-To call any of the supported Google Cloud Services simply add a corresponding client library artifact as a dependency to your project. The following instructions use `google-cloud-storage` as an example (specific instructions can be found in the README of each client).
+To call any of the supported Google Cloud Services simply add a corresponding client library 
+artifact as a dependency to your project. The following instructions use `google-cloud-storage` 
+as an example (specific instructions can be found in the README of each client).
 
 [//]: # ({x-version-update-start:google-cloud-storage:released})
 If you are using Maven, add this to your pom.xml file
 ```xml
-<dependency>
-  <groupId>com.google.cloud</groupId>
-  <artifactId>google-cloud-storage</artifactId>
-  <version>1.51.0</version>
-</dependency>
+  <dependencyManagement>
```

Comment:
You need a version update tag for google-cloud-bom. It can't be inside the google-cloud-storage tag though - nesting isn't supported. Probably the best thing would be to create two xml code blocks.

Reply:
I fixed this one